var/spack/repos/builtin/packages/python/package.py

```diff
@@ -1122,82 +853,317 @@ def setup_dependent_build_environment(self, env, dependent_spec):
+            config_args.append("--with-computed-gotos")
 
-    def test(self):
-        # do not use self.command because we are also testing the run env
-        exe = self.spec["python"].command.name
+        if spec.satisfies("@3.7 %intel"):
+            config_args.append("--with-icc={0}".format(spack_cc))
 
-        # test hello world
-        msg = "hello world!"
-        reason = "test: running {0}".format(msg)
-        options = ["-c", 'print("{0}")'.format(msg)]
-        self.run_test(exe, options=options, expected=[msg], installed=True, purpose=reason)
+        if "+debug" in spec:
+            config_args.append("--with-pydebug")
+        else:
+            config_args.append("--without-pydebug")
 
-        # checks import works and executable comes from the spec prefix
-        reason = "test: checking import and executable"
-        options = ["-c", "import sys; print(sys.executable)"]
-        self.run_test(
-            exe, options=options, expected=[self.spec.prefix], installed=True, purpose=reason
-        )
+        if "+shared" in spec:
+            config_args.append("--enable-shared")
+        else:
+            config_args.append("--disable-shared")
+
+        config_args.append("--without-ensurepip")
+
+        if "+pic" in spec:
+            cflags.append(self.pkg.compiler.cc_pic_flag)
+
+        if "+ssl" in spec:
+            config_args.append("--with-openssl={0}".format(spec["openssl"].prefix))
+
+        if "+dbm" in spec:
+            # Default order is ndbm:gdbm:bdb
+            config_args.append("--with-dbmliborder=gdbm")
+        else:
+            config_args.append("--with-dbmliborder=")
+
+        if "+pyexpat" in spec:
+            config_args.append("--with-system-expat")
+        else:
+            config_args.append("--without-system-expat")
+
+        if "+ctypes" in spec:
+            config_args.append("--with-system-ffi")
+        else:
+            config_args.append("--without-system-ffi")
+
+        if "+tkinter" in spec:
+            config_args.extend(
+                [
+                    "--with-tcltk-includes=-I{0} -I{1}".format(
+                        spec["tcl"].prefix.include, spec["tk"].prefix.include
+                    ),
+                    "--with-tcltk-libs={0} {1}".format(
+                        spec["tcl"].libs.ld_flags, spec["tk"].libs.ld_flags
+                    ),
+                ]
+            )
+
+        # https://docs.python.org/3.8/library/sqlite3.html#f1
+        if spec.satisfies("+sqlite3 ^sqlite+dynamic_extensions"):
+            config_args.append("--enable-loadable-sqlite-extensions")
+
+        if spec.satisfies("%oneapi"):
+            cflags.append("-fp-model=strict")
+
+        if cflags:
+            config_args.append("CFLAGS={0}".format(" ".join(cflags)))
+
+        return config_args
+
+    def install(self, pkg, spec, prefix):
+        with working_dir(self.build_directory):
+            make(*self.install_targets, parllel=False)
```

Comment:
```suggestion
            # See https://github.com/python/cpython/issues/102007
            make(*self.install_targets, parallel=False)
```

Reply:
:point_up: This issue wasn't fixed

Reply:
The most important thing was the typo. That seems not fixed.

Reply:
Honestly didn't even notice it. Amended.